reaktive/src/commonMain/kotlin/com/badoo/reaktive/base/ErrorCallbackExt.kt

```diff
@@ -1,11 +1,15 @@
 package com.badoo.reaktive.base
 
-internal inline fun <T> ErrorCallback.tryCatch(block: () -> T, onSuccess: (T) -> Unit) {
+internal inline fun <T> ErrorCallback.tryCatch(
+    block: () -> T,
+    errorTransformer: (Throwable) -> Throwable = { it },
```

Comment:
The only purpose of this transformer is to create CompositeException. Can we replace it with something like
```kotlin
tryCatch(
    block: ...,
    composeExceptionWith: Throwable? = null,
    onSuccess: ...
)
```

Reply:
I wanted more generic function, so it could be reused later in different case. `composeExceptionWith` is very specific.

Reply:
Then maybe create overloaded version with `composeExceptionWith` parameter?
I don't like all this copy-pasted `{ CompositeException(it, exception) }`.

Reply:
Will do

Reply:
Tried, it clashes with existing version in some circumstances, e.g. if you specify only `block` and `onSuccess` arguments in the following way: `tryCatch({...}) { ... }`.